templates/server.conf.erb

```diff
@@ -47,12 +47,16 @@
 [database]
 name: <%= scope['pulp::db_name'] %>
 seeds: <%= scope['pulp::db_seeds'] %>
+<%# Auth is currently only working on mongo > 2.4 -%>
+<% mongo_version = scope.lookupvar('::mongodb_version') -%>
+<% if mongo_version && scope.function_versioncmp([mongo_version, '2.6']) >= 0 -%>
```

Comment:
nit: you use 2.6 here and 2.6.0 in the puppet code. I'd use the exact same version string.